### ISSUE TITLE
Hints for troubleshooting build?

### ISSUE THREAD

**Issue by spujadas** [skiselev/easy_z80#3]
Hi,

First of all, thanks for the great project, had a great time working through the design!

At this point, I’ve manufactured the PCB (version 1.1), purchased the components as per the BOM, soldered everything, programmed the Flash ROM with RomWBW (EZZ80_std.rom version 3.0.1) and the SPLD with the provided fuse map, and have a jumper across J4 (no SRAM battery for now).
Plugging the board in to a 5V source lights up the LED, I’m reading 5V and 0V on the relevant power pins of the chips, and my signal analyser is showing that the oscillators are up and running and that there appears to be some movement on the chips.
But connecting a serial terminal (115,200 baud, 8 data bits, 1 stop bit, no parity, no flow control) to the primary serial port shows absolutely nothing (the serial cable is tested as working fine with another project).

Would you have any hints as to how to approach troubleshooting the board to work out what I missed or did wrong?

Thanks very much!

**Comment by skiselev** [comment]
Can you please try enabling RTS/CTS flow control in your terminal emulation software?

**Comment by spujadas** [comment]
Thanks for the reply, changed to RTS/CTS flow control but made no difference unfortunately. (Also tried the other flow controls just in case.)

**Comment by skiselev** [comment]
Next, I'd check the /RESET signal on the CPU (should be high) , activity on the control lines and address lines of the CPU, probably starting with /M1, /MREQ, /RD, A0, next check /ROM_CS signal. If all appear to be active, check SIO/2 chip select signal.

**Comment by spujadas** [comment]
Thanks for your guidance.
The /RESET signal on the CPU was alternating between high for 1.376s and low for 43ms, but other than that I wasn’t seeing any activity on the CPU.
I realised that the CPU wasn’t receiving the clock signal (no activity whatsoever on pin 6). No idea why, especially considering that all the other chips were getting this signal.

Anyway, I soldered a wire to connect the oscillator to the CPU, and whilst I still wasn’t getting anything on the serial port, at the very least I started seeing some activity on the pins you mentioned (and the /RESET signal was stable at high)… for a few minutes, and then nothing again despite several resets.
I checked that the CPU was still working using another project, and confirmed it was indeed the case.

At this point, I’m suspecting poor PCB manufacturing or poor soldering on my part (even though nothing jumps out, the connections look fine both physically and electrically), so as fortunately I have 4 more Easy-Z80 PCBs (where I’ve confirmed that pin 6 of the CPU is indeed electrically connected to the output of the oscillator), I think I’ll probably redo everything from scratch.

Long story short, I’ll close this issue as it seems likely that something is actually physically wrong with the project on my end.
Thanks again Sergey for your quick replies and helpful suggestions!
